CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 cmake_minimum_required(VERSION 3.0)
+set(SOVERSION 1)
```

Comment:
Why not just set PROJECT_VERSION (`project(portmidi VERSION 1.0.0)`)?

Reply:
I suppose I could do that. I wanted to keep that consistent with the tags. I started with 235.0 because the last SVN revision was 234 and I wanted to ensure people weren't stuck with packages that wouldn't update.

Reply:
By the way, PortAudio has the same weird situation where the .so version is unrelated to what the release is called.

Reply:
You can have `PROJECT_VERSION` be unrelated to the git tag, but it does cause a weird situation and I'm not sure how other distributions would like that. Not sure what the best approach is, this does work too.

Reply:
I suppose I could version the library 235.2 to be consistent.